bin/cyhy-ip

```diff
@@ -163,6 +179,8 @@ def add(db, owner, cidrs):
     intersections = db.RequestDoc.get_all_intersections(
         cidrs
     )  # intersections from database
+    if has_restricted_ips(cidrs):
+        print("Cannot continue!\nSome addresses are in invalid countries.") # Current countries that are not to be scanned.
```

Comment:
I think you want a `sys.exit(-1)` or something similar here.  Otherwise processing won't stop.

Reply:
Ah, good catch. I thought I added that. Will do.